### DISCUSSION TITLE
Time on SenseCAP_Indicator incorrect after flashing Home Assistant

### DISCUSSION THREAD

**Discussion by eighthill89** [Seeed-Solution/SenseCAP_Indicator_ESP32#44] [Q&A]
Hello community,

I am a complete newbie to SenseCap Indicator. I have successfully flashed the SenseCAP Indicator for integration with Home Assistant. I can see the data from all sensors in Home Assistant. 

But now I have the problem that the time on the SenseCap Indicator is wrong by an hour. The localization is correct. I believe that summer or winter time is not displayed correctly. But I also get the following error in Visual Studio Code after flashing:

I (79356) city: Get time zone...
E (79387) esp-tls-mbedtls: mbedtls_ssl_setup returned -0x7F00
E (79388) esp-tls: create_ssl_handle failed
E (79388) esp-tls: Failed to open new connection
E (79399) city: Connection failed...

I would be very grateful for any help!

Best regards!
Daniel

**Comment by Love4yzp** [comment]
There may be repeat connections.
Was this successful after the reboot?
